x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/selectors.ts

```diff
@@ -142,7 +141,7 @@ export const searchBarIndexPatterns = (state: AlertListState) => state.searchBar
  * query params to use when requesting alert data.
  */
 export const apiQueryParams: (
-  state: AlertListState
+  state: Immutable<AlertListState>
```

Comment:
its not generally necessary to add `Immutable` everywhere, but it can be useful when debugging. i may or may not remove these. they dont hurt anything